src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/DossierServiceTest.kt

```diff
@@ -20,8 +21,9 @@ internal class DossierServiceTest {
   private val revocationOrderService = mockk<RevocationOrderService>()
   private val pdfDocumentGenerator = mockk<PdfDocumentGenerator>()
   private val recallDocumentService = mockk<RecallDocumentService>()
+  private val pdfDecorator = mockk<PdfDecorator>()
 
-  private val underTest = DossierService(revocationOrderService, pdfDocumentGenerator, recallDocumentService)
+  private val underTest = DossierService(revocationOrderService, pdfDocumentGenerator, recallDocumentService, pdfDecorator)
 
   @Test
   fun `get dossier returns part A, license and revocation order as dossier when all present for recall`() {
```

Comment:
You'll love this: add "as numbered pages" to test title?

Reply:
we're not numbering it tho, its mocked.

Reply:
Yeah, but the same is true of every step in this test.  IMO, it's a pretty weak test.  And yes, I wrote the original version.  The component test comes to the rescue fortunately.